File: prak10/PRAK110-2510817210003-MuhammadRaffaAzisha.py
import math # Diperlukan untuk fungsi math.sqrt() (akar kuadrat)
alas = 5
tinggi = 12
sisi_c = math.sqrt((alas ** 2) + (tinggi ** 2))
keliling = alas + tinggi + sisi_c
luas = 0.5 * alas * tinggi

print("Diketahui :")
print(f"Alas = {int(alas)} cm")
print(f"Tinggi = {int(tinggi)} cm\n")

print("Jawab :")
print(f"Sisi A = {int(alas)} cm")
print(f"Sisi B = {int(tinggi)} cm")
print(f"Sisi C = {int(sisi_c)} cm")
print(f"Keliling = {int(keliling)} cm")
print(f"Luas = {int(luas)} cm")

File: prak10/PRAK110-2510817210003-MuhammadRaffaAzisha.c
#include <stdio.h>
#include <math.h> //untuk fungsi sqrt() (akar kuadrat)

int main() {
    double alas = 5.0;
    double tinggi = 12.0;
    double sisi_c;
    double luas;
    double keliling;
    sisi_c = sqrt((alas * alas) +  (tinggi * tinggi));
    keliling = alas + tinggi + sisi_c;
    luas = 0.5 * alas * tinggi;
        
    printf("Diketahui :\n");
    printf("Alas = %.0f cm\n", alas);
    printf("Tinggi = %.0f cm\n\n", tinggi);    
    printf("Jawab :\n");
    printf("Sisi A = %.0f cm\n", alas);
    printf("Sisi B = %.0f cm\n", tinggi);
    printf("Sisi C = %.0f cm\n", sisi_c);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas = %.0f cm\n", luas);
    
    return 0;
}